services/cargohold/src/CargoHold/API.hs

```diff
@@ -167,6 +167,12 @@ sitemap = do
         header "Z-User"
         .&. capture "key"
 
+    document "DELETE" "deleteAsset" $ do
+        Doc.summary "Delete an asset"
+        Doc.parameter Doc.Path "key" Doc.bytes' $
+            Doc.description "Asset key"
+        Doc.response 200 "Asset deleted" Doc.end
```

Comment:
Please also add `Doc.errorResponse Error.assetNotFound` and `Doc.errorResponse Error.unauthorised`.

Reply:
> Please also add Doc.errorResponse Error.assetNotFound and Doc.errorResponse Error.unauthorised.

Well spotted, 👍

Reply:
Also added a few more error responses that I noticed were not documented https://github.com/wireapp/wire-server/commit/f2eb47f0cb4c549991bfb5ea65e2c69b2a4ce440